### PR TITLE
Fix the failing soledad test when invite codes are enabled

### PR DESCRIPTION
**Pull request by ankonym**
When invite codes are required, the soledad sync test will now pass again because an invite code will be generated for the user creation test.

### CODE REVIEW

**Comment by ankonym** [comment]
Note: This depends on [this commit](https://github.com/Alster-Hamburgers/leap_web/commit/0ef9035c62358ec3f74de3591a9580e5bd426422) getting merged into leap platform to work.

File: tests/helpers/bonafide_helper.rb [hunk 1]
```diff
@@ -40,6 +48,15 @@ def assert_create_user
     return user
   end
 
+  def generate_invite_code
+    `cd /srv/leap/webapp/ && sudo -u leap-webapp RAILS_ENV=production bundle exec rake generate_invites[1]`.gsub(/\n/, "")
+  end
+
+  # TODO: @ankonym delete invite code after test run
```
Comment (varac) [inline tests/helpers/bonafide_helper.rb]: please don't include empty definitions and todo items ;)

File: puppet/modules/couchdb [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit 1f583d9c9157390850a7737630f60832ced82374
```
Comment (varac) [inline puppet/modules/couchdb]: why is this submodule update needed ?

**Comment by ankonym** [comment]
Okay, this is now cleaned up to only contain the changed file. Yay!

**Pull request merged**